src/PooltogetherGovernor.sol

```diff
@@ -70,6 +70,16 @@ contract PooltogetherGovernor is
   {
     return GovernorTimelockCompound.supportsInterface(interfaceId);
   }
+  /// @dev We override this for it to show up in our scopelint spec
+
+  function propose(
```

Comment:
Why did we need this now when we previously didn't 🤔

Reply:
We don't need to do this, but the reason I did was to integrate the tests better with `scopelint spec`. It makes it easy to see exactly what test cases we cover. The output of the command in a future looks like the below.

![Screenshot from 2023-07-11 17-53-18](https://github.com/ScopeLift/pooltogether-governor-upgrade/assets/12705146/64c5458b-733b-4624-b756-b840eb69ed54)